storage/pebble/registers.go

```diff
@@ -74,22 +89,55 @@ func (s *Registers) Store(
 		}
 	}
 
-	err := batch.Commit(pebble.Sync)
+	err = batch.Commit(pebble.Sync)
 	if err != nil {
 		return fmt.Errorf("failed to commit batch: %w", err)
 	}
 
 	return nil
 }
 
-// TODO: Finish implementation after deciding prefixes and/or badger use
-
 // LatestHeight Gets the latest height of complete registers available
 func (s *Registers) LatestHeight() (uint64, error) {
-	panic("not implemented")
+	return s.heightLookup(latestHeightKey())
 }
 
 // FirstHeight first indexed height found in the store, typically root block for the spork
 func (s *Registers) FirstHeight() (uint64, error) {
-	panic("not implemented")
+	return s.heightLookup(firstHeightKey())
+}
+
+func (s *Registers) heightLookup(key []byte) (uint64, error) {
+	res, closer, err := s.db.Get(key)
+	defer closer.Close()
+	if err != nil {
+		return 0, err
+	}
+	return binary.BigEndian.Uint64(res), nil
+}
+
+func (s *Registers) SetFirstHeight(height uint64) error {
+	return s.indexHeightWithKey(firstHeightKey(), height)
+}
+
+func (s *Registers) SetLatestHeight(height uint64) error {
```

Comment:
I think we agreed to remove this method, no?

https://github.com/onflow/flow-go/pull/4671#discussion_r1320020294

Reply:
wanted to sync about it here: https://discord.com/channels/613813861610684416/1151865078203822153/1151894667542142976
If `Store` is called in batches, it should not update `latestHeight` until all batches are stored.

Reply:
Yeah, the latest height is set within the same batch. 

There is just one single batch with multiple writes, N writes are for storing N updated registers, and 1 write is for updating the latest height. Right?

Reply:
Yes, this method is no longer needed if we update height in store batch.